openmmtools/forcefactories.py

```diff
@@ -86,8 +86,19 @@ def replace_reaction_field(reference_system, switch_width=1.0*unit.angstrom,
 # RESTRAIN ATOMS
 # =============================================================================
 
-def restrain_atoms(thermodynamic_state, sampler_state, topology,
-                   atoms_dsl, sigma=3.0*unit.angstroms):
+def restrain_atoms_by_dsl(thermodynamic_state, sampler_state, topology, atoms_dsl, **kwargs):
```

Comment:
We will need to make a change to YANK to use this correctly in the `pipeline` correct? If that issue is not already open, this is just to serve as a reminder.